lightning/src/ln/channelmanager.rs

```diff
@@ -11408,6 +11477,7 @@ where
 			(11, probing_cookie_secret, option),
 			(13, claimable_htlc_onion_fields, optional_vec),
 			(14, decode_update_add_htlcs, option),
+			(15, awaiting_invoice_flag, option),
```

Comment:
Updated code to get awaiting_invoice_flag info from pending_outbound_payments instead of serialising it in [2566089](https://github.com/lightningdevkit/rust-lightning/commit/2566089ec0b127e55febc934c43677a3d00d1716)
Thanks for the suggestion!